tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.ScenarioTests.ps1

```diff
@@ -0,0 +1,345 @@
+Import-Module -Name AzureRM.Bootstrapper
+$RollupModule = 'AzureRM'
+
+function Remove-InstalledProfile { 
+    $installedProfiles = Get-AzureRmProfile
+    if ($installedProfiles -ne $null)
+    {
+        foreach ($profile in $installedProfiles)
+        {
+            Write-Host "Removing profile $profile"
+            Uninstall-AzureRmProfile -Profile $profile -Force -ErrorAction Stop
+        }
+    }
+}
+
+Describe "A machine with no profile installed can install profile" {
+    # Using Install-AzureRmProfile
+    Context "New Profile Install - Latest" {
+        # Launch the test in a new powershell session
+        # Arrange
+        # Uninstall previously installed profiles
+        Remove-InstalledProfile       
+
+        # Create a new PS session
+        $session = New-PSSession
+
+        # Act
+        # Install latest version
+        Invoke-Command -Session $session -ScriptBlock { Install-AzureRmProfile -Profile 'Latest' } 
+
+        # Assert 
+        It "Should return Latest Profile" {
+            Invoke-Command -Session $session -ScriptBlock { Get-AzureRmProfile } | Should Be 'Latest'
+        }
+
+        # Clean up
+        Invoke-Command -Session $session -ScriptBlock { Uninstall-AzureRmProfile -Profile 'Latest' -Force }
+        Remove-PSSession -Session $session
+    } 
+
+    # Using Use-AzureRmProfile
+    Context "New Profile Install - 2016-09" {
+        # Arrange
+        # Uninstall previously installed profiles
+        Remove-InstalledProfile
+
+        # Create a new PS session
+        $session = New-PSSession
+
+        # Act
+        # Install profile '2016-09'
+        Invoke-Command -Session $session -ScriptBlock { Use-AzureRmProfile -Profile '2016-09' -Force }
+
+        # Assert
+        It "Should return 2016-09" {
+            Invoke-Command -Session $session -ScriptBlock { Get-AzureRmProfile } | Should Be '2016-09'
+        }
+
+        # Clean up
+        # Invoke-Command -Session $session -ScriptBlock { Uninstall-AzureRmProfile -Profile '2016-09' -Force -ea SilentlyContinue }
+        Remove-PSSession -Session $session
+    }
+
+}
+
+Describe "Add: A Machine with a Profile installed can install latest profile" {
+    Context "Profile 2016-09 already installed" {
```

Comment:
TODO:  Need to add a scenario for installing 2 named profiles side by side and verifying that we cna select each